accounts/safe7579/src/zkEmailRecovery/modules/RecoveryModuleBase.sol

```diff
@@ -1,71 +1,49 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.23;
 
-import {ERC7579ExecutorBase} from "@rhinestone/modulekit/src/Modules.sol";
+import { ERC7579ExecutorBase } from "@rhinestone/modulekit/src/Modules.sol";
 
 abstract contract RecoveryModuleBase is ERC7579ExecutorBase {
```

Comment:
imo its better to not implement the virtual functions. could lead to issues where other recovery modules that inherit this forget to overwrite it